src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -204,7 +204,7 @@ const JurisdictionTable = (props: JurisdictionSelectorTableProps) => {
       />,
       <NodeCell key={`${node.model.id}-jurisdiction`} node={node} baseUrl={baseUrl} />,
       node.model.node.attributes.structureCount,
-      <SelectedJurisdictionsCount
+      <ConnectedSelectedJurisdictionsCount
```

Comment:
Just curious why you chose to connect `SelectedJurisdictionsCount` to the redux store?  It seems like its parent component is already connected and could have used the selector and fed it into `SelectedJurisdictionsCount` as a prop.

Reply:
I figured it was cleaner to pass the filters this way since the result if the `selectedLeafNodes` as one of the filters to pass to the `getSelectedNodesUnderParentNode`